### DISCUSSION TITLE
2D resolution

### DISCUSSION THREAD

**Discussion by wpotrzebowski** [SasView/sasview#2499] [General]
As discussed at the magnetic SANS workshop in Lund, 2023 we were wondering how 2D resolution is treated. Particularly, we were wondering if dqx and dqy read from the file are treated by the resolution function.

**Comment by wpotrzebowski** [comment]
https://github.com/SasView/sasmodels/blob/master/sasmodels/resolution2d.py

**Comment by astellhorn** [comment]
Separate question would be:
How a 3rd dimension of resolution could be treated?

**Reply by astellhorn** [reply]
An example from diffraction for multi-dimensional resolution treatment: Acta Cryst. (1975). A31, 507 ("On the Resolution of Slow-Neutron Spectrometers., IV. The Triple-Axis Spectrometer Resolution Function, Spatial Effects Included")

**Comment by wpotrzebowski** [comment]
Testing main with 6 on 2D data set that has dQpara and dQperp. I would expect "Smearing" to say "Use dQ data" but it is empty. Is it a bug?

**Comment by astellhorn** [comment]
Idea for the next canSAS meeting:
Include a discussion about 2D resolution functions:
- How to store it?
- How to read it into SasView?
- How to calculate 2D/3D resolution functions?
- TOF resolution
- How to parametrize asymmetric resolution / non-gaussians (e.g. TOF-case)
